cmd/scw/testdata/test-all-usage-rdb-instance-create-usage.golden

```diff
@@ -6,16 +6,16 @@ USAGE:
   scw rdb instance create [arg=value ...]
 
 ARGS:
-  name                Name of the instance
-  engine              Database engine of the database (PostgreSQL, MySQL, ...)
-  user-name           Name of the user created when the instance is created
-  password            Password of the user
-  node-type           Type of node to use for the instance (DEV1-S, ...)
-  [is-ha-cluster]     Whether or not High-Availability is enabled
-  [disable-backup]    Whether or not backups are disabled
-  [tags.{index}]      Tags to apply to the instance
-  [organization-id]   Organization ID to use. If none is passed will use default organization ID from the config
-  [region=fr-par]     Region to target. If none is passed will use default region from the config (fr-par | nl-ams)
+  name                 Name of the instance
+  engine               Database engine of the database (PostgreSQL, MySQL, ...)
+  user-name            Name of the user created when the instance is created
+  password             Password of the user
+  node-type=db-dev-s   Type of node to use for the instance (DEV1-S, ...) (db-dev-s | db-dev-m | db-dev-l | db-dev-xl | db-gp-xs | db-gp-s | db-gp-m | db-gp-l | db-gp-xl)
```

Comment:
We did not do this for instance. Should we wait for caching so we can update list based on API result ?

Reply:
I think we should set the default first and modify this code when the cache will be available.